language/de-DE/lib_joomla.ini

```diff
@@ -312,11 +312,71 @@ JLIB_FORM_CHANGE_USER="Benutzer auswählen"
+JLIB_FORM_FIELD_PARAM_CALENDAR_FORMAT_DESC="TDas Datumsformat, das verwendet wird. This is in the format used by PHP to specify date string formats (see below). If no format argument is given, '%Y-%m-%d' is assumed (giving dates like '2008-04-16')."
+JLIB_FORM_FIELD_PARAM_CALENDAR_FORMAT_LABEL="Format"
+JLIB_FORM_FIELD_PARAM_CHECKBOX_MULTIPLE_DESC="Erlabt die Auswahl mehrerer Optionen."
+JLIB_FORM_FIELD_PARAM_CHECKBOX_MULTIPLE_LABEL="Mehrfachauswahl"
+JLIB_FORM_FIELD_PARAM_CHECKBOX_MULTIPLE_VALUES_DESC="The options of the list."
+JLIB_FORM_FIELD_PARAM_CHECKBOX_MULTIPLE_VALUES_KEY_LABEL="Key"
+JLIB_FORM_FIELD_PARAM_CHECKBOX_MULTIPLE_VALUES_LABEL="Checkbox Werte"
+JLIB_FORM_FIELD_PARAM_CHECKBOX_MULTIPLE_VALUES_VALUE_LABEL="Wert"
+JLIB_FORM_FIELD_PARAM_EDITOR_BUTTONS_HIDE_DESC="Die Buttons in dieser Liste (getrennt duch Komma) werden nicht gezeigt."
+JLIB_FORM_FIELD_PARAM_EDITOR_BUTTONS_HIDE_LABEL="Versteckte Buttons"
+JLIB_FORM_FIELD_PARAM_EDITOR_HEIGHT_DESC="Legt die Höhe des Editorfensters fest (in px) - Standardwert ist 250px."
+JLIB_FORM_FIELD_PARAM_EDITOR_HEIGHT_LABEL="Höhe"
+JLIB_FORM_FIELD_PARAM_EDITOR_SHOW_BUTTONS_DESC="Should the buttons being shown."
+JLIB_FORM_FIELD_PARAM_EDITOR_SHOW_BUTTONS_LABEL="Zeige Buttons"
+JLIB_FORM_FIELD_PARAM_EDITOR_WIDTH_DESC="Legt die Breite des Editorfensters fest (in px) - Standardwert ist 100%."
+JLIB_FORM_FIELD_PARAM_EDITOR_WIDTH_LABEL="Breite"
+JLIB_FORM_FIELD_PARAM_IMAGELIST_DIRECTORY_DESC="The filesystem path to the directory containing the image files to be listed."
+JLIB_FORM_FIELD_PARAM_IMAGELIST_DIRECTORY_LABEL="Verzeichnis"
+JLIB_FORM_FIELD_PARAM_INTEGER_FIRST_DESC="This value is the lowest on the list." 
+JLIB_FORM_FIELD_PARAM_INTEGER_FIRST_LABEL="First"
+JLIB_FORM_FIELD_PARAM_INTEGER_LAST_DESC="This value is the highest on the list."
+JLIB_FORM_FIELD_PARAM_INTEGER_LAST_LABEL="Letzter"
+JLIB_FORM_FIELD_PARAM_INTEGER_STEP_DESC="Each option will be the previous option incremented by this integer, starting with the first value until the last value is reached."
+JLIB_FORM_FIELD_PARAM_INTEGER_STEP_LABEL="Schrittweite"
+JLIB_FORM_FIELD_PARAM_LIST_MULTIPLE_DESC="Ermöglich die Auswahl mehrerer Optionen."
+JLIB_FORM_FIELD_PARAM_LIST_MULTIPLE_LABEL="Mehrfachauswahl"
+JLIB_FORM_FIELD_PARAM_LIST_MULTIPLE_VALUES_DESC="Die Optionen der Liste."
+JLIB_FORM_FIELD_PARAM_LIST_MULTIPLE_VALUES_LABEL="List Values"
+JLIB_FORM_FIELD_PARAM_LIST_MULTIPLE_VALUES_VALUE_LABEL="Wert"
+JLIB_FORM_FIELD_PARAM_LIST_MULTIPLE_VALUES_NAME_LABEL="Name"
+JLIB_FORM_FIELD_PARAM_MEDIA_IMAGE_CLASS_DESC="The class which is added to the image (src tag)."
+JLIB_FORM_FIELD_PARAM_MEDIA_IMAGE_CLASS_LABEL="CSS Klasse des Bildes"
+JLIB_FORM_FIELD_PARAM_MEDIA_PREVIEW_DESC="Shows or hides the preview of the selected image."
+JLIB_FORM_FIELD_PARAM_MEDIA_PREVIEW_INLINE="Inline"
+JLIB_FORM_FIELD_PARAM_MEDIA_PREVIEW_LABEL="Vorschau"
+JLIB_FORM_FIELD_PARAM_MEDIA_PREVIEW_TOOLTIP="Tooltip"
+JLIB_FORM_FIELD_PARAM_MEDIA_HOME_LABEL="HomeVerzeichnis"
```

Comment:
```suggestion
JLIB_FORM_FIELD_PARAM_MEDIA_HOME_LABEL="Benutzerverzeichnis"
```

Reply:
Ich finde Benutzerverzeichnis falsch - HomeVerzeichnis auch ...  
Ich weiss aber nicht, was es sein soll. Das Verzeichnis, in dem alle media abgelegt werden? Media-Verzeichnis?